packages/curvenote/src/session/session.ts

```diff
@@ -71,6 +78,7 @@ export class Session implements ISession {
     if (this.SITE_URL !== DEFAULT_SITE_URL) {
       this.log.warn(`Connecting to Site at: "${this.SITE_URL}".`);
     }
+    this.JOURNALS_URL = STAGING_JOURNALS_API_URL;
```

Comment:
We are hard coding to the staing environment.

In order to properly be able to test in staging, we are going to need to bring up a staging CDN mirroring prod -- which we have not done yet. Then we can select the URl base don the token as we do with the SITE_URL. cc @rowanc1 

Otherwise we need a mechanisms to set the api and other URLS. It seems like `SessionOptions` provides for this but I cannot see anywhere were we can pass those in, `--apiUrl` is not an argument on the cli yet -- @fwkoch was that the intention? this is a bit cumbersome for testing too, should we do something via environment variable instead allowing it to be set in the background? this would make both local and staging testing easier.